extras/rootless/containerd-rootless.sh

```diff
@@ -102,12 +102,11 @@ if [ -z $_CONTAINERD_ROOTLESS_CHILD ]; then
 			fi
 		fi
 	fi
-	if [ -z $mtu ]; then
+	if [ -z "$mtu" ]; then
 		mtu=1500
 	fi
 
-	_CONTAINERD_ROOTLESS_CHILD=1
-	export _CONTAINERD_ROOTLESS_CHILD
+	export _CONTAINERD_ROOTLESS_CHILD=1
```

Comment:
This bashism may not work with some /bin/sh implementation

Reply:
Yep... you are right... 

Not quite sure why we are exporting this one btw.

Anyhow, will reverse this line.